src/client/styles/components/AccountPage.scss

```diff
@@ -90,15 +90,15 @@
 
   .tabbed ul[role="tablist"] {
     &> li {
-      border-color: #0576D3;
+      border-color: $accountBlue;
```

Comment:
Is there a reason you're using these $variables instead of the DS color variables?

Reply:
Yes, mostly because I'm not sure if we are okay with completely moving to the DS colors and breakpoint values. I was going to make that a separate ticket to first document these different and make sure everyone knows and is on board with the changes.